metabase/metabase
# Do not load google auth when not enabled

Fixes https://github.com/metabase/metabase/issues/24737

How to test:
- Run Metabase without Google Auth enabled
- There should be no request to `https://accounts.google.com/gsi/client`
- Go to Admin -> Settings -> Authentication -> Google and enable Google auth with the correct `Client ID`
- There should be a request to `https://accounts.google.com/gsi/client`
- Make sure it is possible to log in with Google credentials
